### PR TITLE
[REF] Allow this repo to be a pre-commit hook

### PR DESCRIPTION
**Pull request by fernandahf**
With these changes, this repo will be used as a hook in [pre-commit](https://github.com/pre-commit/pre-commit).

### CODE REVIEW

**Comment by fernandahf** [comment]
For record:

The tests worked fine!

Screenshot 2022-12-19 at 15 42 54

Screenshot 2022-12-19 at 15 43 11

cc @moylop260

**Comment by fernandahf** [comment]
@markdrago 

Could you take a look at this, please?

Those little changes allow using this repo as the hook of https://github.com/pre-commit/pre-commit

More info: 

https://pre-commit.com/#new-hooks

**Comment by moylop260** [comment]
@markdrago

What do you think?

File: README.md [hunk 1]
```diff
@@ -64,6 +64,17 @@ Additionally PgSanity will read SQL from stdin if it is not given any parameters
     $ echo $?
     0
 
+# # pre-commit
```
Comment (markdrago) [inline README.md:67]: Please fix the markdown syntax here: "# #" -> "##"

Comment (fernandahf) [reply README.md]: @markdrago 

Fixed. Thanks!

**Pull request merged**